.claude/commands/simplify.md

```
@@ -0,0 +1,119 @@
+  if (condition1) {
+    return condition2 ? 'a' : 'b';
+  }
+  return condition3 ? 'c' : 'd';
+}
+const result = getResult();
+```
+
+---
+
+**Before:**
+```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)